### PR TITLE
Searching for an element in a linked list C++

### PR DESCRIPTION
**Pull request by geoffgeorgein**
Contributed by - Geoff George <geoffgeorgethommana@gmail.com>

### CODE REVIEW

File: Data Structures/Linked List/linked list search.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+ 
+
+int main()
+{
+   
+    Node* head = NULL;
+    int x = 21;
+ 
+    14->21->11->30->10 */
```
Comment (harshit-sharma-gits) [inline Data Structures/Linked List/linked list search.cpp:45]: I believe this is a comment, please make it so

File: Data Structures/Linked List/linked list search.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    push(&head, 10);
+    push(&head, 30);
+    push(&head, 11);
+    push(&head, 21);
+    push(&head, 17);
+ 
+    
+    search(head, 11) ? cout << "Yes" : cout << "No";
```
Comment (harshit-sharma-gits) [inline Data Structures/Linked List/linked list search.cpp:53]: Can you be more elaborative on what the output is telling us?

File: Data Structures/Linked List/linked list search.cpp [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#include <bits/stdc++.h>
```
Comment (harshit-sharma-gits) [inline Data Structures/Linked List/linked list search.cpp:1]: Please take a look at #12 
You have to add your name, email in the specified format in the first line as comment

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
I have added comments in your file, please resolve them

**Comment by geoffgeorgein** [comment]
Added the required changes

File: Data Structures/Linked List/linked list search.cpp (hunk 3 above)
Comment (harshit-sharma-gits) [reply Data Structures/Linked List/linked list search.cpp:1]: Add you name, email in first line as well @geoffgeorgein, see issue #13

**Review by harshit-sharma-gits — APPROVED** [review]

**Pull request merged**